static/extensions/minidogg/sitebuilder.js

```diff
@@ -0,0 +1,306 @@
+class siteBuilder {
+    constructor (runtime) {
+        this.runtime = runtime
+        this.siteHtml = ""
+        this.siteWindow;
+    }
+    getInfo() {
+      return {
+        id: 'sitebuilder',
+        name: 'Site Builder',
+        //colors
+        color1: '#7e69beff',
+        color2: '#3a286f',
+        docsURI: 'https://github.com/minidogg/my-penguinmod-extensions/blob/88ff87b8fff13b4e601415d94b201f12c6d475fc/site%20builder/docs/home.md',

```

Comment:
```suggestion
        // docsURI: 'https://github.com/minidogg/my-penguinmod-extensions/blob/88ff87b8fff13b4e601415d94b201f12c6d475fc/site%20builder/docs/home.md',
```
If this is still work and progress, might as well comment it out.